docs/run-core-node/tier-1-orgs.mdx

```diff
@@ -3,39 +3,45 @@ title: Tier 1 Organizations
 sidebar_position: 120
 ---
 
-To help with Stellar’s decentralization, the most reliable and advanced Stellar teams join the ranks of “Tier 1 Organizations.” These organizations run three validators, coordinate any changes to their quorumsets, and hold themselves to a higher standard of uptime and responsiveness.
+To help with Stellar’s decentralization, the most advanced teams building on Stellar run validators and strive to join the ranks of “Tier 1 organizations.”
 
-SDF works closely with Tier 1 Orgs to ensure the health of the network, maintain good quorum intersection, and build in redundancy to minimize network disruptions. This guide outlines what it takes to be a Tier 1 Org.
+Tier 1 organizations are the organizations that are members of minimal quorums in the core of the Stellar network. This is an emergent property of the Stellar network that depends on the configurations of all the validators in the Stellar network.
```

Comment:
This is the one sentence I get stuck on.  I think I trip over "minimal quorums" (what are those? why are there multiple?  what makes them minimal?), and over the "core of the Stellar network" (not Stellar Core, but something else?) and "emergent property" (that one I can look up, but still it's a difficult concept) and "depends on the configurations of all validators" (_all_ validators?  hard to wrap my head around).  Maybe skip this?  Or move it later, after the next two paragraphs that are a bit easier to grep?

Reply:
What about the following?

Remember that the Stellar network consists of organizations that each run validators, and each organization decides for itself, by configuring a quorum set, which and how many other organizations it requires agreement from in order to commit to a particular new ledger. Tier 1 organizations are a group of organizations that, due to the fact that most other organizations require agreement from them, bear the safety and liveness of the Stellar network on their shoulders[^1].

[^1] This can be defined precisely, but is besides the point of this document

Reply:
That's a larger discussion, but I think that it might make sense to switch from talking about "quorum sets" (which is bad because a quorum set is not a set of quorums) to talking about "agreement requirements". Especially since we use "proof of agreement".

Reply:
I think that new bit is fantastic!  Very clear.  Re: "quorum sets," I'm not opposed to changing it ("trust configurations"?), but I think you're correct that it would take a fair amount of finding and replacing, and the change may not stick since people are used to saying "quorum sets," and it can be hard to change usage once a phrase is widespread.